ethd

```diff
@@ -953,7 +953,7 @@ envmigrate() {
     return 0
   fi
 
-  ALL_VARS=( COMPOSE_FILE FEE_RECIPIENT WITHDRAWAL_CREDENTIALS EL_NODE GRAFFITI DEFAULT_GRAFFITI NETWORK MEV_BOOST MEV_RELAYS MEV_MIN_BID \
+  ALL_VARS=( COMPOSE_FILE FEE_RECIPIENT EL_NODE GRAFFITI DEFAULT_GRAFFITI NETWORK MEV_BOOST MEV_RELAYS MEV_MIN_BID \
```

Comment:
Please add the new CL_OBOL_NODE and EL_OBOL_NODE here, so their value is kept during ethd update

Reply:
done